target/linux/lantiq/image/Makefile

```diff
@@ -241,9 +261,12 @@ endef
 TARGET_DEVICES += audiocodes_mp-252
 
 define Device/arcadyan_arv4510pw
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := ARV4510PW
+  DEVICE_ALT0_VENDOR := Wippies (BeWAN)
+  DEVICE_ALT0_MODEL := BWA iBox V1.0
```

Comment:
This is an odd one, not sure about the ALT0 values.

Reply:
I'd go for
```
DEVICE_ALT0_VENDOR := Wippies
DEVICE_ALT0_MODEL := BeWan iBox v1.0
```

but without clear precedence.

Reply:
You might also set `Wippies/Elisa` for ALT0_VENDOR, but do not believe that any solution is more correct than the other here. Don't put time into it ...

Reply:
I have not found any reference to the "Elisa" string. I would drop it as it is unclear if that is meant as a vendor or model name.